jenkins.sh

```diff
@@ -40,7 +40,7 @@ export RAILS_ENV=test
 bundle install --path "${HOME}/bundles/${JOB_NAME}" --deployment --without development
 
 # Uncomment if this app uses a database
-bundle exec rake db:drop db:create db:schema:load
+bundle exec rake db:drop db:create db:schema:load db:test:prepare
```

Comment:
We shouldn't need to add this.  We haven't had to for any other repos. `db:schema:load` will load the schema defined in `db/schema.rb` into the database (assuming it's been checked in - it could well do nothing if `db/scheam/rb` doesn't exist).